en/Getting started/Sync your notes across devices.md

```diff
@@ -2,6 +2,8 @@ Syncing means keeping your notes up to date across your devices, such as your la
 
 The easiest way to sync your notes between your devices is to use [[Introduction to Obsidian Sync|Obsidian Sync]]. If you already have an Obsidian Sync subscription, see how to [[Set up Obsidian Sync]].
 
+Note that using multiple syncing services at the same time (e.g., Obsidian Sync _and_ Dropbox) may cause data loss, corruption, and other issues. [[Obsidian Sync and third-party services||Find out more about using Obsidian Sync with other services here.]]
```

Comment:
```suggestion
> [!important]
> Using multiple syncing services at the same time may cause data loss, corruption, and other issues—for example using Obsidian Sync _and_ Dropbox together. For more information on using Obsidian Sync with other services, refer to [[Obsidian Sync and third-party services]].
```

- I think this warrants an important callout?
- Prefer using the title of the page as the link text, and avoid using full sentences as link text.
- Prefer using "for example" or "such as" instead of e.g. ([Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)).